fix_txresponse.sh

```
@@ -0,0 +1,76 @@
+#!/bin/bash
+
+# Fix all remaining TxResponse creations
+file="crates/helium-baseapp/src/lib.rs"
+
+# Function to fix a TxResponse at a given line
+fix_txresponse() {
+    local start_line=$1
+    local temp_file=$(mktemp)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)